12-offer-encoding.md

```diff
@@ -220,9 +220,9 @@ The human-readable prefix for offers is `lno`.
     1. type: 20 (`offer_quantity_max`)
     2. data:
         * [`tu64`:`max`]
-    1. type: 22 (`offer_node_id`)
+    1. type: 22 (`offer_issuer_id`)
```

Comment:
The renaming is incomplete, "offer_node_id" still appears many times. You haven't renamed it in invoice requests and invoices.

Reply:
Yes, there were a few places, I've done a sweep and fixed those.  Thanks!